CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+-   Added CHANGELOG.md
+-   Added new functionality for saving seed phrase in-app so that it can be used for recovery.
+
+### Changed
+
+-   Changed Terms and Conditions screen to new UI.
+
+## [1.1.7]
+
+### Added
+
+-   Initial implementation.
```

Comment:
This formulation suggests to me that there are no releases before this.

Maybe this?

```suggestion
-   Last release without changelog.
```

Reply:
I also think there should be a changelog for the legacy wallet.

Reply:
I was just thinking

```
# Legacy Wallet

## [3.2.0]

Last release without a changelog.
```

Assuming that 3.2.0 is the latest release.

Reply:
But actually this changelog isn't going to be merged into that branch, so I guess we could just add the changelog there if any more work is done there.